SwiftLeedsWidget/SwiftLeedsMediumWidgetView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SwiftLeedsMediumWidgetView.swift
+//  SwiftLeedsWidgetExtension
+//
+//  Created by karim ebrahim on 09/09/2022.
+//
+
+import SwiftUI
+import WidgetKit
+
+struct SwiftLeedsMediumWidgetView: View {
+    
+    // MARK: - Pivate Properties
+    
+    private let slot: Schedule.Slot
+    private let timeFontSize: CGFloat = 10
+    private let titleFontSize: CGFloat = 14
+    private let detailsFontSize: CGFloat = 14
+    private let logoImageWidth: CGFloat = 45
+    private let logoImageHeight: CGFloat = 45
+    
+    // MARK: - Init
+    
+    init(slot: Schedule.Slot) {
+        self.slot = slot
+    }
+    
+    // MARK: - Body View
+    
+    var body: some View {
+        buildSlotView(for: slot)
+    }
+}
+
+// MARK: - View Builders
+
+extension SwiftLeedsMediumWidgetView {
+    @ViewBuilder
+    private func buildSlotView(for slot: Schedule.Slot) -> some View {
+        if let activity = slot.activity {
+            slotView(time: slot.startTime, speaker: activity.title, details: activity.description!)
+        }
+
+        if let presentation = slot.presentation {
+            let speakers = presentation.speakers
+            let firstSpeakerName = speakers.first?.name  ?? ""
+            slotView(time: slot.startTime, speaker: firstSpeakerName, details: presentation.title)
+        }
+    }
+    
+    @ViewBuilder
+    private func slotView(time: String, speaker: String = "", details: String) -> some View {
+        ZStack {
+            Color.background
+            HStack {
+                contentView(time: time, speaker: speaker, details: details)
+            }
+        }
+    }
+    
+    @ViewBuilder
+    private func contentView(time: String, speaker: String = "", details: String) -> some View {
+        VStack(alignment: .leading) {
+            HStack {
+                logoView
+                Spacer()
+                ZStack {
+                    Rectangle()
+                        .fill(Color.cellForeground.opacity(0.1))
+                        .frame(width: 100, height: 30, alignment: .center)
+                        .cornerRadius(14)
+                    
+                    Text("Up Next")
```

Comment:
Use the verbatim initialiser, i.e. Text(verbatim: "Up Next") as it's static text.